cli/cmd/project/partitions.go

```diff
@@ -26,7 +26,8 @@ func PartitionsCmd(ch *cmdutil.Helper) *cobra.Command {
 				model = args[1]
 			}
 
-			if !local && !cmd.Flags().Changed("project") && len(args) <= 1 && ch.Interactive {
+			if !local && !cmd.Flags().Changed("project") && !cmd.Flags().Changed("model") &&
+				len(args) <= 1 && ch.Interactive {
```

Comment:
I don't think it makes sense to bring `!cmd.Flags().Changed("model")` in here. Because we need to know the project to query. So if a project is not specified, we _need_ to either prompt for it or error. 

Again, it's a bit difficult to formalize, but I think "don't prompt when flags are specified" is a principle for optional (and unrelated) fields, not for necessary/required fields (where we need to either prompt or error).